cpp/src/strings/convert/convert_datetime.cu

```diff
@@ -433,6 +433,19 @@ struct datetime_formatter {
   const int32_t* d_offsets;
   char* d_chars;
 
+  __device__ auto units_base(timestamp_units units) -> int64_t
+  {
+    switch (units) {
+      case timestamp_units::hours: return 24L;
+      case timestamp_units::minutes: return 1440L;   // 24*60
+      case timestamp_units::seconds: return 86400L;  // 24*60*60
+      case timestamp_units::ms: /**/ return 86400000L;
```

Comment:
curious. why `/**/`?

Reply:
For alignment of the `return`s

Reply:
Are you kidding me? You missed one, BTW.

Reply:
Think of the opportunity cost of spending time aligning code.